classes/submission/reviewer/ReviewerAction.inc.php

```diff
@@ -56,8 +56,9 @@ function confirmReview($request, $reviewAssignment, $submission, $decline, $emai
 			if ($emailText) $email->setBody($emailText);
 			$email->send($request);
 
+			$currentDate = Core::getCurrentDate();
 			$reviewAssignment->setDeclined($decline);
-			$reviewAssignment->setDateConfirmed(Core::getCurrentDate());
+			$reviewAssignment->setDateConfirmed($currentDate);
```

Comment:
I don't think this change does anything.

Reply:
True, it's an artifact of the completed reviews correction. I'll revert it.